Tests/Admin/BreadcrumbsBuilderTest.php

```diff
@@ -133,113 +133,4 @@ public function testChildGetBreadCrumbs()
-            'link'
-        )->willReturn('My child class');
-        $labelTranslatorStrategy->getLabel(
-            'my_child_class_name_my_action',
-            'breadcrumb',
-            'link'
-        )->willReturn('My action');
-        if ($action == 'create') {
-            $labelTranslatorStrategy->getLabel(
-                'my_class_name_create',
-                'breadcrumb',
-                'link'
-            )->willReturn('create my object');
-            $admin->trans('create my object', array(), null)->willReturn('Créer mon objet')->shouldBeCalled();
-            $menu->addChild('Créer mon objet', array())->willReturn($menu);
-        }
-        $childAdmin = $this->prophesize('Sonata\AdminBundle\Admin\AbstractAdmin');
-        $childAdmin->getLabelTranslatorStrategy()->willReturn($labelTranslatorStrategy->reveal());
-        $childAdmin->getClassnameLabel()->willReturn('my_child_class_name');
-        $childAdmin->hasRoute('list')->willReturn(false);
-        $childAdmin->trans('My child class', array(), null)->willReturn('Ma classe fille');
-        $childAdmin->trans('My action', array(), null)->willReturn('Mon action');
-        $childAdmin->getCurrentChildAdmin()->willReturn(null);
-        $childAdmin->hasSubject()->willReturn(false);
-
-        $admin->trans('My class', array(), null)->willReturn('Ma classe');
-        $admin->hasRoute('list')->willReturn(true);
-        $admin->isGranted('LIST')->willReturn(true);
-        $admin->generateUrl('list')->willReturn('/myadmin/list');
-        $admin->getCurrentChildAdmin()->willReturn(
-            $action == 'my_action' ? $childAdmin->reveal() : false
-        );
-        if ($action == 'list') {
-            $admin->isChild()->willReturn(true);
-            $menu->setUri(false)->shouldBeCalled();
-        }
-        $request = $this->prophesize('Symfony\Component\HttpFoundation\Request');
-        $request->get('slug')->willReturn('my-object');
-
-        $admin->getIdParameter()->willReturn('slug');
-        $admin->hasRoute('edit')->willReturn(false);
-        $admin->getRequest()->willReturn($request->reveal());
-        $admin->hasSubject()->willReturn(true);
-        $admin->getSubject()->willReturn('my subject');
-        $admin->toString('my subject')->willReturn('My subject');
-        $admin->getLabelTranslatorStrategy()->willReturn(
-            $labelTranslatorStrategy->reveal()
-        );
-        $admin->getClassnameLabel()->willReturn('my_class_name');
-
-        $menu->addChild('Ma classe', array(
-            'uri' => '/myadmin/list',
-        ))->willReturn($menu->reveal());
-        $menu->addChild('My subject')->willReturn($menu);
-        $menu->addChild('My subject', array('uri' => null))->willReturn($menu);
-        $menu->addChild('Ma classe fille', array('uri' => null))->willReturn($menu);
-        $menu->addChild('Mon action', array())->willReturn($menu);
-
-        $breadcrumbsBuilder->buildBreadCrumbs($admin->reveal(), $action);
-    }
```

Comment:
I mean it is less tests, but rewriting them to test them throught the public method would be too much work. So either simply remove them, like this, or play with reflection to make the method public just during tests.